packages/styled-components/src/components/primitives/Image/Image.styled.ts

```diff
@@ -1,9 +1,17 @@
+import { StyledCssProps } from 'components/Components.styled.types';
 import styled from 'styled-components';
+import { customCss } from 'utils/customCss';
 
-export const StyledSource = styled.source`
+type Props = StyledCssProps;
+
+export const StyledSource = styled.source<StyledCssProps>`
   user-select: none;
+
+  ${(props) => customCss(props)}
 `;
 
-export const StyledImage = styled.img`
+export const StyledImage = styled.img<StyledCssProps>`
   user-select: none;
+
+  ${(props) => customCss(props)}
```

Comment:
Looks identical except the tag. Maybe we can create a single StyledImage which will render an img tag as default and we can provide as="source" for the source tag.